src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/UWP/PopupRenderer.uwp.cs

```diff
@@ -283,6 +300,7 @@ protected virtual void Dispose(bool disposing)
 				Control = null;
 
 				Closed -= OnClosing;
```

Comment:
BTW, should it be ```Closing -= OnClosing```? I see that we do ```Closing += OnClosing``` in this file

Reply:
This appears to the same feedback as the other thread.